### ISSUE TITLE
Detaching Expokit not giving Libraries for IOS

### ISSUE THREAD

**Issue by Rutulpatel7077** [expo/expo#2063]
### Environment

Environment:
  OS: macOS High Sierra 10.13.6
  Node: 9.4.0
  Yarn: 1.7.0
  npm: 5.6.0
  Watchman: 4.9.0
  Xcode: Xcode 9.4.1 Build version 9F2000
  Android Studio: 3.1 AI-173.4697961

Packages: (wanted => installed)
  expo: ^27.0.0 => 27.1.0
  react: 16.3.1 => 16.3.1
  react-native: https://github.com/expo/react-native/archive/sdk-27.0.0.tar.gz => 0.55.2

### Steps to Reproduce

1. Use simple react-native app created by expo
2. `exp detach`
3. `react-native link`
4. I tried `pod install` in ios folder
5. Still No Libraries folder found! ?

### Expected Behavior
It should generate ios folder with all the Libraries of RN

### Actual Behavior
It is not generating any of that folders.

**Comment by terribleben** [comment]
Hi there! Are you referring to this directory?

https://github.com/facebook/react-native/tree/master/Libraries

You can find your copy of react native, including all of its native iOS and Android code, by looking under `your-detached-project/node_modules/react-native`.

**Comment by Rutulpatel7077** [comment]
@terribleben  Thanks a lot...!

Should I just copy and paste all the Libraries to my ios folder?

**Comment by terribleben** [comment]
No need to copy and paste! They are already included in your iOS project.

If you look at the contents of `your-project/ios/Podfile`, you will see that the React Native iOS dependency is linked to the same copy:

```rb
  pod 'React',
    :path => "../node_modules/react-native",
    :inhibit_warnings => true,
    ...
```

So when you run `pod install` you are already getting the correct iOS code from `react-native` (including `Libraries`).

**Comment by Rutulpatel7077** [comment]
Excellent! @terribleben  

I have seen that folders but now understood I might have to solve some Xcode errors.
I thought the problem is with Libraries.

 Sorry for bugging.

**Comment by terribleben** [comment]
No problem, good luck with your project :)

**Comment by zilan830** [comment]
I have also met this problem. I used `expokit`. 
`create-react-native-app my-project
  cd my-project
  expo eject
`
Then I use  `react-native-pdf`. It also use [`rn-fetch-blob`.](https://www.npmjs.com/package/react-native-pdf)
 But  I can not run my project. The Error: `undefined is not an object (evaluatin 'RNFetchBolb.DocumentDir)`. 
A document teach me to link package manually, but I can not find `Libraries `.
I think it may be because of `expokit`.
Can you tell me how to deal with this problem.
Thx!

**Comment by timoisalive** [comment]
I'm still struggling with this...

I've ejected to ExpoKit and would need to use RCTLinkingManager. Sure I can find it in the node_modules but what do I have to do to be able to import and use it in the AppDelegate.m!?

**Comment by timoisalive** [comment]
Ok, I got it.

First of all, I had to open the `xcworkspace` file instead of `xcodeproj` file, to have the CocoaPod libraries available in XCode.

Then I had to manually add RCTLinkingIOS into the `Podfile`:

```pod 'React',
    :path => "../node_modules/react-native",
    :inhibit_warnings => true,
    :subspecs => [
      "RCTLinkingIOS",
      ...
    ]```

And finally run `pod install` in the `ios` folder.

I think that’s all… :slightly_smiling_face:

**Comment by mikeyyyzhao** [comment]
My project used to use expo but we ejected(because of Stripe). Now we're trying to use react-native-notifications but in the installation instructions, it says to manually link the RCTPushNotification into Libraries which still does not exist in our project. Is there any way to create it? I saw some posts sayings "react-native link" will solve the issue but that hasn't created the library groups inside
